### DISCUSSION TITLE
有需要修改的嘛

### DISCUSSION THREAD

**Discussion by 245633131** [Meloong-Git/PCL#5204] [自定义主页]
file:///C:/baidunetdiskdownload/%E6%B7%B7%E6%B2%8C%E7%A7%91%E6%8A%80%E7%A9%BA%E5%B2%9B/%E6%B7%B7%E6%B2%8C%E7%A9%BA%E5%B2%9B1.0/%E8%9B%8B%E4%BB%94[%E5%86%85%E6%B5%8B%E7%AB%AF]v0.1/PCL/Custom.xaml

**Comment by Ad-closeNN** [comment]
?这是你本地的链接，你需要将其上传到 GitHub

只需将你的 Custom.xaml 重命名为.txt 并拖放到 "Add a comment" 下面的框框并等待 Uploading xxx 字样消失后点击右下角的 "Comment" 即可

**Comment by LinQingYuu** [comment]
file:// 为本地文件链接，如果要收录预设需要稳定的订阅地址和相关介绍

有关主页标准与要求参考 #2528，详细指导参考 #4483

**Comment by CreeperIsASpy** [comment]
Additon List:
- [ ] 截图
- [ ] 可用的地址

~~Congratulations, you've got none of them!~~
